Dockerfile

```diff
@@ -2,10 +2,9 @@ FROM node:6.3.1
 
 MAINTAINER Daniel J. Pritchett <daniel@clearfunction.com>
```

Comment:
Might could have updated the maintainer to be Dr. Mo ;)

Reply:
Yeah, I saw it after the merge, definitely should do that...